client/components/button/docs/example.jsx

```diff
@@ -59,6 +59,12 @@ var Buttons = React.createClass( {
 						<Button primary disabled >Primary disabled button</Button>
 					</div>
 					<div className="design-assets__button-row">
+						<Button secondary >Secondary button</Button>
```

Comment:
Incorrect spacing on this line and the following ones: there should be no spaces between the last attribute/value in an opening tag and the `>` character — i.e. correct form is `<Button secondary>`